app/workers/stale_issue_marker.rb

```diff
@@ -0,0 +1,71 @@
+class StaleIssueMarker
+  include Sidekiq::Worker
+  sidekiq_options :queue => :miq_bot_glacial, :retry => false
+
+  include SidekiqWorkerMixin
+
+  # If an issue/pr has any of these labels, it will not be marked as stale or closed
+  PINNED_LABELS    = ['pinned'].freeze
+
+  STALE_LABEL_NAME = 'stale'.freeze
+  STALE_ISSUE_MESSAGE = <<-EOS
+This issue has been automatically marked as stale because it has not been updated for at least 3 months.
```

Comment:
Although stale_date now has 6 months, this string (and the next constant) still have 3 months as text.